Mathlib/Algebra/Order/Group/Defs.lean

```diff
@@ -892,7 +892,7 @@ theorem div_le_inv_mul_iff [CovariantClass α α (swap (· * ·)) (· ≤ ·)] :
 #align div_le_inv_mul_iff div_le_inv_mul_iff
 
 --  What is the point of this lemma?  See comment about `div_le_inv_mul_iff` above.
-@[to_additive, simp]
+@[to_additive, simp] -- do we intentionally not have this simp for the additive declaration?
```

Comment:
unanswered question

Reply:
This is just a mistake from #869. Fixed now.